src/azul/drs.py

```diff
@@ -269,21 +268,38 @@ def get_object(self,
         depends on the access method specified.
         """
         if access_id is None:
-            return self._get_object(object_id, access_method=access_method)
+            return self._get_object(drs_uri, access_method)
         else:
-            return self._get_object_access(object_id, access_id, access_method=access_method)
+            return self._get_object_access(drs_uri, access_id, access_method)
 
-    def _get_object(self, object_id: str, access_method: AccessMethod) -> str:
-        url = self.url / object_id
+    @classmethod
```

Comment:
Does this have to be a class method?

Reply:
And be public?

Reply:
If the motivation is in the chained PR, make it a protected instance method in this PR and then make the necessary modification in the other PR.